server/models/PaymentMethod.js

```diff
@@ -308,9 +331,13 @@ export default function(Sequelize, DataTypes) {
       }
 
       // If there is no monthly limit, the user needs to be an admin of the collective that owns the payment method
-      if (!this.monthlyLimitPerMember && !user.isAdmin(this.CollectiveId)) {
+      if (
+        !this.monthlyLimitPerMember &&
+        !user.isAdmin(this.CollectiveId) &&
+        this.type !== 'manual'
+      ) {
         throw new Error(
-          "You don't have enough permissions to use this payment method (you need to be an admin of the collective that owns this payment method)",
+          `You don't have enough permissions to use this payment method (you need to be an admin of the collective that owns this payment method)`,
```

Comment:
we better use single quotes when there's no variable to be inserted in the string template.

Reply:
good catch. I'll use double quotes here because of the `don't`.